modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -44,7 +55,7 @@ $ oc -n openshift-ingress-operator edit ingresscontroller/default
         protocol: PROXY
       type: HostNetwork
 ----
-* If your Ingress Controller uses the NodePortService endpoint publishing strategy type, set the `spec.endpointPublishingStrategy.nodePort.protocol` subfield to `PROXY`:
+* If your Ingress Controller uses the `NodePortService` endpoint publishing strategy type, set the `spec.endpointPublishingStrategy.nodePort.protocol` subfield to `PROXY`:
```

Comment:
can add the following for supporting the Private endpoint publishing strategy type:
If your Ingress Controller uses the `Private` endpoint publishing strategy type, set the `spec.endpointPublishingStrategy.private.protocol` subfield to `PROXY`:
.Sample `private` configuration to `PROXY`
spec:
  endpointPublishingStrategy:
    type: Private
    private:
      protocol: PROXY

Reply:
:heavy_check_mark: